plugin/hover.py

```diff
@@ -107,25 +107,20 @@ def symbol_actions_content(self, point: int) -> str:
             actions = []
             for link_kind in link_kinds:
                 if self.best_session('{}Provider'.format(link_kind.lsp_name)):
-                    command = 'lsp_run_command_from_point'
-                    args = {
-                        'command_name': link_kind.subl_cmd_name,
-                        'point': point,
-                    }
-                    link = make_command_link(command, link_kind.label, args)
+                    link = make_command_link(link_kind.subl_cmd_name, link_kind.label, {'point': point})
```

Comment:
By setting `"mdpopups.debug": 3` in my Packages/User/Preferences.sublime-settings I can see the following html output (formatted a bit):

```html
<p class="actions">
    <a href='subl:lsp_symbol_definition {&quot;point&quot;: 939}'>
        Definition
    </a>
    <a href='subl:lsp_symbol_definition {&quot;side_by_side&quot;: true, &quot;point&quot;: 939}' class='icon'>
        ◨
    </a>
     | 
    <a href='subl:lsp_symbol_references {&quot;point&quot;: 939}'>References</a>
     |
    <a href='subl:lsp_symbol_rename {&quot;point&quot;: 939}'>Rename</a>
</p>
```

This doesn't work on the first view, but using `sublime.log_commands(True)` I can actually see it being called. I have no idea why this doesn't work.

Reply:
It looks like the `subl:` command triggered from the popup runs on the active view rather than on the view in which the popup was triggered.

It also looks like the `point` argument is not passed to the command so with this refactored code, the navigation to the symbol in the current view is also broken (if the cursor is not on the symbol while clicking the `Definition` link)

Reply:
> It also looks like the `point` argument is not passed to the command so with this refactored code, the navigation to the symbol in the current view is also broken (if the cursor is not on the symbol while clicking the `Definition` link)

Actually, this issue is due to the `is_enabled` check preventing the command from running:
```py
    def is_enabled(self, event: Optional[dict] = None) -> bool:
        return super().is_enabled(event) and is_at_word(self.view, event)
```

Reply:
> Actually, this issue is due to the `is_enabled` check preventing the command from running:

I've pushed a fix for that.

I think there are still issues with:
 - ST calling `subl:` command on the wrong view if a popup is triggered for non-active view
 - Sometimes there are issues with things not working on the first view after opening ST as you've mentioned